poetry/packages/vcs_dependency.py

```diff
@@ -1,3 +1,5 @@
+from poetry.vcs import git
```

Comment:
```suggestion
from ...vcs import git
```

Reply:
Hello @matemax,

I'm not a big fan of relative imports. "Explicit is better than implicit." and is better to read. Absolute imports are also used (most if the time) in the poetry code.

Reply:
Hello @finswimmer ,
It is ony suggestion. I try use these code for me and noticed relative and not relative import in one file.

Reply:
The rule I have personally is if the import comes from the same module/package, it should be a relative import otherwise it should be an absolute import. An example of this is https://github.com/python-poetry/poetry/blob/master/poetry/masonry/builders/builder.py